packages/@sanity/vision/src/components/ParamsEditor.js

```diff
@@ -0,0 +1,66 @@
+import React, {PropTypes} from 'react'
+import classNames from 'classnames'
+import ReactCodeMirror from 'react-codemirror'
+require('codemirror/mode/javascript/javascript')
```

Comment:
How do I import when there is no need for an variable? It just hooks up to the codemirror namespace i think